File: entity-admin-android/app/src/main/java/com/example/entityadmin/model/Subscriber.kt
package com.example.entityadmin.model

data class Subscriber(
    val id: String, // Assuming String ID, adjust if Long or other type from backend
    val name: String?, // Nullable to handle backend inconsistencies
    val email: String?, // Nullable to handle backend inconsistencies
    val nfcCardUid: String? // Nullable if not always present
)

File: mobile/ios/entity-admin/EntityAdminIOS/Models/Subscriber.swift
// entity-admin-ios/EntityAdminIOS/Models/Subscriber.swift
import Foundation

struct Subscriber: Codable, Identifiable {
    let id: String // Assuming String ID from backend, adjust if Int/UUID
    let name: String
    let email: String
    let nfcCardUid: String? // Optional, as per Android model

    // If your backend uses different keys, define CodingKeys:
    // enum CodingKeys: String, CodingKey {
    //     case id
    //     case name
    //     case email
    //     case nfcCardUid = "nfc_card_uid" // Example for backend key "nfc_card_uid"
    // }
}
